packages/vite/src/node/ssr/ssrTransform.ts

```diff
@@ -257,6 +257,8 @@ interface Visitors {
   onDynamicImport: (node: Node) => void
 }
 
+const IsNodeInPatternWeakMap = new WeakMap<_Node, boolean>()
```

Comment:
**question:** What it the goal of a `PascalCase` named variable?
I'm usually used to `CONSTANT_CASE` or `camelCase`

Reply:
This type comes from previous content. There are two Node types that are imported and one is aliased to _Node.
If it breaks the naming convention, I would like to issue another PR to fix it.

Reply:
I think we can use `isNodeInPatternWeakMap` here, what do you mean by this name coming from previous content?

Reply:
sorry I didn't see it right, I will change it.